books/RayTracingInOneWeekend.html

```diff
@@ -1884,37 +1996,31 @@
 True Lambertian Reflection
 ---------------------------
 <div class='together'>
-The rejection method presented here produces random points in the unit ball offset along the surface
-normal. This corresponds to picking directions on the hemisphere with high probability close to the
-normal, and a lower probability of scattering rays at grazing angles. This distribution scales by
-the $\cos^3 (\phi)$ where $\phi$ is the angle from the normal. This is useful since light arriving
-at shallow angles spreads over a larger area, and thus has a lower contribution to the final color.
-
-However, we are interested in a Lambertian distribution, which has a distribution of $\cos (\phi)$.
-True Lambertian has the probability higher for ray scattering close to the normal, but the
-distribution is more uniform. This is achieved by picking random points on the surface of the unit
-sphere, offset along the surface normal. Picking random points on the unit sphere can be achieved by
-picking random points _in_ the unit sphere, and then normalizing those.
+Using a diffuse model that scatters vectors evenly about the hemisphere produces a nice and soft
+diffuse model, but we can definitely do better. A more accurate representation of real diffuse
+objects is the Lambertian distribution, which has a scattering distribution that is propertional to
+$\cos (\phi)$, where $\phi$ is the angle from the normal. This means that a ray has a higher
+probability of scattering toward the normal than it does scattering toward the horizon. This
+distribution is less uniform than pure hemispherical scattering, but does a better job of modeling 
+objects in the real world. We can create this distribution by picking random points on the surface
+of the unit sphere, and then offsetting them along the surface normal.
 
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
-    inline vec3 random_in_unit_sphere() {
-        ...
-    }
-
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-    vec3 random_unit_vector() {
-        return unit_vector(random_in_unit_sphere());
-    }
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [random-unit-vector]: <kbd>[vec3.h]</kbd> The random_unit_vector() function]
+There are two unit radius spheres tangent to the hit point $p$ of a surface. These two spheres have
+a center of $(\mathbf{P} + \mathbf{n})$ and $(\mathbf{P} - \mathbf{n})$, where $\mathbf{n}$ is the
+normal of the surface. The sphere with a center at $(\mathbf{P} - \mathbf{n})$ is considered
```

Comment:
How about this instead:
“Since a sphere's normal vector points away from the sphere center, the inside tangent sphere's center is ... and the outside tangent sphere's center is ... ”